src/mdesktopaction.h

```diff
@@ -0,0 +1,43 @@
+/***************************************************************************
+** This file was derived from the MDesktopAction implementation in the
+** libmeegotouch library.
+**
+** Copyright (C) 2021 Chupligin Sergey (NeoChapay) <neochapay@gmail.com>
+**
+** This library is free software; you can redistribute it and/or
+** modify it under the terms of the GNU Lesser General Public
+** License version 2.1 as published by the Free Software Foundation
+** and appearing in the file LICENSE.LGPL included in the packaging
+** of this file.
+**
+****************************************************************************/
+
+#ifndef MDESKTOPACTION_H
+#define MDESKTOPACTION_H
+
+#include <mlite-global.h>
+#include <QString>
+
+class MDesktopEntryPrivate;
+
+class MLITESHARED_EXPORT MDesktopAction
+{
+
+public:
+    MDesktopAction(const QString &fileName, const QString &action);
+    virtual ~MDesktopAction();
+
+    QString name() const;
+    QString nameUnlocalized() const;
+    QString getExec() const;
```

Comment:
Most (if not all) getters in this library don't use the **get** prefix, I would expect this one also to be called `exec()` like in `MDesktopEntry`, for the sake of consistency.